src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsServiceExtensions.cs

```diff
@@ -38,5 +37,28 @@ public static bool IsRegularOrDocumentationComment(this ISyntaxFactsService synt
         {
             syntaxFacts.GetPartsOfAssignmentStatement(statement, out left, out _, out right);
         }
+
+        public static SyntaxNode Unparenthesize(
```

Comment:
Do you have a test with interspersed trivia?

Reply:
Yes.  This was jsut a move of an existing piece of code (part of hte simplifier).  That code has been tested in the past with different comment forms for parenthesized expressions.